idaes_examples/mod/power_gen/soc_dynamic_flowsheet.py

```diff
@@ -0,0 +1,1477 @@
+                "CO2":(1-0.35)/(1-0.2074)*0.0003,
+                "N2":(1-0.35)/(1-0.2074)*0.7732,
+                "Ar":(1-0.35)/(1-0.2074)*0.0092,
+            }
+            self._set_gas_port(
+                self.sweep_recycle_mix.recycle, F=2750, T=1020, P=1.2e5, y=recycle_comp, fix=False)
+
+        self.sweep_blower.initialize(outlvl=outlvl, solver=solver, optarg=optarg)
+        propagate_state(self.sweep01)
+        
+        self.feed_recycle_mix.initialize(outlvl=outlvl, solver=solver, optarg=optarg)
+        self.sweep_recycle_mix.initialize(outlvl=outlvl, solver=solver, optarg=optarg)
+
+        propagate_state(self.feed03)
+        propagate_state(self.sweep03)
+        self.sweep_heater.default_initializer(
+            solver=solver, solver_options=optarg, output_level=outlvl
+        ).initialize(model=self.sweep_heater)
+        self.feed_heater.default_initializer(
+            solver=solver, solver_options=optarg, output_level=outlvl
+        ).initialize(model=self.feed_heater)
+        # self.sweep_heater.initialize(outlvl=outlvl, solver=solver, optarg=optarg)
+        # self.feed_heater.initialize(outlvl=outlvl, solver=solver, optarg=optarg)
+        propagate_state(self.feed04)
+        propagate_state(self.sweep04)
+        
+        if fuel_cell_mode:
+            self.soc_module.potential_cell.fix(0.902)
+            self.soc_module.initialize(
+                current_density_guess=0, #4000, Don't know why the initialization has so much trouble with the real value
+                temperature_guess=980,
+                outlvl=outlvl,
+                solver=solver,
+                optarg=optarg
+                )
+            self.sweep_recycle_split.split_fraction[:, "recycle"].fix(0.50)
+            self.feed_recycle_split.split_fraction[:, "recycle"].fix(0.01)
+        else:
+            self.soc_module.potential_cell.fix(1.3)
+            self.soc_module.initialize(
+                current_density_guess=0,
+                temperature_guess=1020.15,
+                outlvl=outlvl,
+                solver=solver,
+                optarg=optarg
+            )
+            self.sweep_recycle_split.split_fraction[:, "recycle"].fix(0.50)
+            self.feed_recycle_split.split_fraction[:, "recycle"].fix(0.5)
+
+        propagate_state(self.ostrm01)
+        self.sweep_recycle_split.initialize(outlvl=outlvl, solver=solver, optarg=optarg)
+        propagate_state(self.ostrm02)
+        propagate_state(self.ostrm03)
+        
+        propagate_state(self.hstrm01)
+        self.feed_recycle_split.initialize(outlvl=outlvl, solver=solver, optarg=optarg)
+        propagate_state(self.hstrm02)
+        propagate_state(self.hstrm03)
+
+        self.sweep_exchanger.default_initializer(
+            solver=solver, solver_options=optarg, output_level=outlvl
+        ).initialize(model=self.sweep_exchanger)
+        # self.sweep_exchanger.initialize(outlvl=outlvl, solver=solver, optarg=optarg)
```

Comment:
Same question for the three commented lines here

Reply:
Forgot to remove the old versions when updating them to the new initialization format. Will take care of that.